tools/CoqMakefile.in

```diff
@@ -177,10 +179,16 @@ COQEXTRAFLAGS?=
 COQCHKEXTRAFLAGS?=
 COQDOCEXTRAFLAGS?=
 
+ifneq ($(EXPLICIT_COQLIB),)
```

Comment:
I use `EXPLICIT_COQLIB` to pass -coqlib to coqc only if it was passed to coq_makefile / in the _CoqProject, but we could just always pass it.